DEV.md

```diff
@@ -1,8 +1,11 @@
-Documentation for core contributors
-===================================
+# Documentation for core contributors
 
-Installing Transifex cli and Sphinx
-------------------------------------
+## RST substitutions
+
+* Use RST substitutions : http://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#default-substitutions
```

Comment:
@Gustry Maybe we should put one or both of those links instead ? 
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#substitutions
https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#substitution-definitions

Reply:
Ok, je vais merger et changer.